app/src/main/res/layout/profile_edit_fragment.xml

```diff
@@ -87,15 +88,15 @@
         style="?android:attr/borderlessButtonStyle"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:background="@color/color_def_white"
+        android:background="@color/component_color_profile_edit_activity_profile_rename_button_color"
```

Comment:
Ditto as earlier comment for another file.

Reply:
updated